config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   get "/healthcheck", to: proc { [200, {}, ["OK"]] }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  post "/batch", to: proc { [202, {}, [{}.to_json]] }
+  get "/batch/:id", to: proc { [200, {}, [{}.to_json]] }
```

Comment:
If `/check` and `/batch` both create jobs (for example when being asynchronous), I think it would be better if the GET variation is `GET /job/:id` since really we're getting information about a job.